rmw_connext_shared_cpp/src/shared_functions.cpp

```diff
@@ -555,6 +574,8 @@ destroy_node(const char * implementation_identifier, rmw_node_t * node)
   node->data = nullptr;
   rmw_free(const_cast<char *>(node->name));
   node->name = nullptr;
+  rmw_free(const_cast<char *>(node->namespace_));
```

Comment:
didn't look up the complete `node` struct here, but what's the difference between the `node` and the `node_handle` is this context?